server/cpho/views/infobase_export.py

```diff
@@ -0,0 +1,220 @@
+    ModelColumn,
+    ModelToSheetWriter,
+)
+from phac_aspc.rules import test_rule
+
+from cpho.models import Benchmarking, Indicator, IndicatorDatum, TrendAnalysis
+from cpho.util import get
+
+indicator_columns = [
+    ModelColumn(Indicator, "id"),
+    ModelColumn(Indicator, "name"),
+    ChoiceColumn(Indicator, "category"),
+    ChoiceColumn(Indicator, "topic"),
+    ModelColumn(Indicator, "detailed_indicator"),
+    ModelColumn(Indicator, "sub_indicator_measurement"),
+    ModelColumn(Indicator, "measure_text"),
+    ModelColumn(Indicator, "title_overall"),
+    ModelColumn(Indicator, "table_title_overall"),
+    ModelColumn(Indicator, "impact_text"),
+    ModelColumn(Indicator, "general_footnotes"),
+    ModelColumn(Indicator, "main_source_english"),
+    ModelColumn(Indicator, "other_relevant_sources_english"),
+    ModelColumn(Indicator, "title_sex"),
+    ModelColumn(Indicator, "table_title_sex"),
+    ModelColumn(Indicator, "title_sex_2"),
+    ModelColumn(Indicator, "table_title_sex_2"),
+    ModelColumn(Indicator, "title_age"),
+    ModelColumn(Indicator, "table_title_age"),
+    ModelColumn(Indicator, "title_age_2"),
+    ModelColumn(Indicator, "table_title_age_2"),
+    ModelColumn(Indicator, "title_province_territory"),
+    ModelColumn(Indicator, "table_title_province_territory"),
+    ModelColumn(Indicator, "title_province_territory_2"),
+    ModelColumn(Indicator, "table_title_province_territory_2"),
+    ModelColumn(Indicator, "title_living_arrangement"),
+    ModelColumn(Indicator, "table_title_living_arrangement"),
+    ModelColumn(Indicator, "title_education_household"),
+    ModelColumn(Indicator, "table_title_education_household"),
+    ModelColumn(Indicator, "title_income_quintiles"),
+    ModelColumn(Indicator, "table_title_income_quintiles"),
+    ModelColumn(Indicator, "title_trend"),
+    ModelColumn(Indicator, "table_title_trend"),
+    ModelColumn(Indicator, "visual_description_trend"),
+    ModelColumn(Indicator, "x_axis_trend"),
+    ModelColumn(Indicator, "y_axis_trend"),
+    ModelColumn(Indicator, "trend_footnotes"),
+    ModelColumn(Indicator, "title_benchmark"),
+    ModelColumn(Indicator, "table_title_benchmark"),
+    ModelColumn(Indicator, "x_axis_benchmark"),
+    ModelColumn(Indicator, "benchmarking_footnotes"),
+    ModelColumn(Indicator, "benchmarking_sources_english"),
+    # quintiles
+    ModelColumn(Indicator, "g1"),
+    ModelColumn(Indicator, "g2_lower"),
+    ModelColumn(Indicator, "g2_upper"),
+    ModelColumn(Indicator, "g3_lower"),
+    ModelColumn(Indicator, "g3_upper"),
+    ModelColumn(Indicator, "g4_lower"),
+    ModelColumn(Indicator, "g4_upper"),
+    ModelColumn(Indicator, "g5"),
+]
+
+
+class IndicatorSheetWriter(AbstractSheetWriter):
+    sheet_name = "indicator"
+
+    def get_column_configs(self):
+        return indicator_columns
+
+
+indicator_data_columns = [
+    ModelColumn(IndicatorDatum, "id"),
+    CustomColumn("indicator_id", lambda x: x.indicator_id),
```

Comment:
CustomColumn("indicator_id"

search returns 6 instances of these, i think these are copy/paste errors.
We need to give them the right column names